docs/en/index.rst

```diff
@@ -148,6 +148,18 @@ Tests for the existence of an element that satisfies the given predicate.
         return $value === 'first';
     }); // true
 
+find
+----
+
+Returns the first element of this collection that satisfies the given predicate.
+
+.. code-block:: php
+    $collection = new Collection(['first', 'second', 'third']);
```

Comment:
Can you please use an example where the element, that is searched for, is twice or more in the collection? It would emphasize the chosen behavior.

Reply:
Sure, done